transport/http/handler.go

```
@@ -117,6 +118,7 @@ func (h handler) callHandler(responseWriter *responseWriter, req *http.Request,
 		Body:            req.Body,
 		BodySize:        int(req.ContentLength),
 	}
+	treq.CallerPeerAddrPort, _ = netip.ParseAddrPort(req.RemoteAddr)
```

Let's not ignore err here. At least we can log it as a Debug entry.